client/doc.go

```diff
@@ -0,0 +1,42 @@
+/*
+Package client provides a library for all client-side CONIKS operations.
+
+Introduction
+
+One crucial component of the CONIKS key management system are the clients
+verifying the cryptographic proofs returned by the CONIKS server as part of
+the registration, lookup and monitoring protocols. client implements the
+operations performed by CONIKS clients during these protocols. This document
+outlines each of these protocols from a client perspective.
+
+Registration
+
+- The user registers her username with the CONIKS client.
+
+- The client generates a new public-private key pair for this username,
+and stores the keys on the device.
+
+- The client sends a registration request
+        reg_req = (username, key)
+to the server (may be validated by a registration bot; see
```

Comment:
Thanks for the clarification @masomel!
I think, if this file describes the actual (client-)protocol implementation it should at least mention the fact that the client (currently) sends a registration request to the bot (or, indirectly to the server but *through* the bot).
I opened an issue as @arlolra suggested (https://github.com/coniks-sys/coniks-go/issues/91)